pages/index.js

```diff
@@ -201,8 +190,74 @@ function HomePage() {
+        <Container className="text-center py-6 pb-20" wide>
+          <Heading level="h3" typeStyle="heading-lg" className="mb-4">
+            Supported by
+          </Heading>
+          <Paragraph className="mt-3 max-w-2xl mx-auto sm:mt-4 pb-4">
+            The following companies support us by letting us use their products for free. Interested in supporting us too?
+            <TextLink href="mailto:info@asyncapi.io" target="_blank">
+              Email us
+            </TextLink> for more info.
+          </Paragraph>
+          <SupportUs className="mt-4" showSupportBanner={false} />
+        </Container>
+        <Container wide>
+          <div className="text-center">
+            <Heading
+              level="h2"
+              typeStyle="heading-lg"
+            >
+              Latest Updates
+            </Heading>
+            <Paragraph typeStyle="body-md" className="max-w-2xl mx-auto mt-3">
+              Get a glimpse of latest news, events, and blog posts. Want to publish a blog post? We love community stories.
+              <TextLink href="https://github.com/asyncapi/website/issues/new?template=blog.md" target="_blank">
+                Submit yours!
+              </TextLink>
+            </Paragraph>
+          </div>
+          <div className="lg:flex flex-row mt-4">
+            <div className="text-center lg:text-left lg:w-1/5 pt-4">
+              <Heading level="h4" typeStyle="heading-md-semibold">
+                From the blog
+              </Heading>
+              <Paragraph typeStyle="body-md" className="mt-4">
+                Check out these articles written by community members
+              </Paragraph>
+              <div className="mt-4">
+                <TextLink href="/blog" className="mt-4">
+                  Read all blog posts
+                  <ArrowRight className="inline w-6" />
+                </TextLink>
+              </div>
+            </div>
+            <NewsroomBlogPosts />
+          </div>
+          <div className="lg:flex flex-row mt-4">
+            <div className="text-center lg:text-left lg:w-1/5 pt-4">
+              <Heading level="h4" typeStyle="heading-md-semibold">
+                Latest News
+              </Heading>
+              <Paragraph typeStyle="body-md" className="mt-4">
+                Read about what people are saying about AsyncAPI
+              </Paragraph>
+              <div className="mt-4">
+                <TextLink href="https://twitter.com/AsyncAPISpec" className="mt-4" target="_blank">
+                  Follow us on Twitter
+                  <ArrowRight className="inline w-6" />
+                </TextLink>
+              </div>
+            </div>
+            <div className="text-center lg:text-left lg:w-4/5 lg:flex flex-row justify-evenly mt-4">
+              <NewsroomArticle />
+              <div className="rounded-md px-6 pt-4 border-2 border-gray-200 mt-4 mx-auto lg:mt-0" style={{width:'fit-content'}}>
```

Comment:
Tbh, I really felt bad when I saw that there is no class in the v1 of TailwindCSS describing the property of `width: fit-content`. I have given it inline style here but it should be added as a utility class.

Reply:
But we will switch to the `v3`, so it won't be problem, am I right 😄